### DISCUSSION TITLE
test data: add more column?

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#606] [Ideas]
Greetings @yesoreyeram 

Could you please add maybe gender, date of birth and geolocation under an address node in this sample data?

https://github.com/yesoreyeram/grafana-infinity-datasource/blob/main/testdata/users.json

Thanks!

**Comment by yesoreyeram** [comment]
May be. What's the use case?

**Reply by yosiasz** [reply]
when trying to demo things to people POC for time series, geomap etc

**Reply by yosiasz** [reply]
one stop shop

**Reply by yesoreyeram** [reply]
Yea. Planning something. 🙂
